_posts/2021-03-03-cockpit-239.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Cockpit 239
+author: lis
+date: 2021-03-03
+tags: cockpit linux
+slug: cockpit-239
+category: release
+summary: Cockpit with restyled updates page
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).  We release regularly.
+
+Here are the release notes from Cockpit version 239.
+
+### Machines: Allow editing disk cache mode
+
+It's now possible to edit the cache mode for existing virtual machine
+images.  This was previously only possible when creating new images.  In
+order to edit the cache mode, any machines using the volume need to be
+powered off.
+
+### Packaging: Removed ./configure options for distribution specific packages
+
+The previous --with-vdo-package, --with-nfs-client-package,
```

Comment:
I'd drop "previous" here. (I realize that I wrote the original)

The options would look better as `code` formatting, as otherwise -- is turned into a long dash. I suggest to copy my original markdown, which also has two links which got lost here. (Edit the description in https://github.com/cockpit-project/cockpit/pull/15415 to get that)

Also, let's move this paragraph to the bottom, to have the UI changes together.